README.md

```diff
@@ -592,7 +592,7 @@ This projects is written in ES2015 and compiled using Babel. The source can be f
 
 To add changes fork (if needed) and clone the project.
 
-```sh
+```bash
```

Comment:
bash and sh are both valid syntax hightlighters, and an alias for the same linguist Shell grammar used in the GitHub Markdown. https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
Can you make sure it's consistent across the readme file please? There are a few more \```bash instances in the file, but most code blocks aren't tagged. Could you tag all of them please with either \```bash or \```sh, whichever one you prefer?

Reply:
sh is fine, I think I did that by mistake :) Will change it back.

Reply:
Will fix the rest and squash the commit! thanks for pointing that out.